diff --git a/packages/libdb/build.sh b/packages/libdb/build.sh
index 1111111111111111111111111111111111111111..2222222222222222222222222222222222222222 100644
--- a/packages/libdb/build.sh
+++ b/packages/libdb/build.sh
@@ -8,6 +8,6 @@ TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION=18.1.40
 TERMUX_PKG_REVISION=5
-TERMUX_PKG_SRCURL=https://fossies.org/linux/misc/db-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SRCURL=https://download.oracle.com/berkeley-db/db-${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=0cecb2ef0c67b166de93732769abdeba0555086d51de1090df325e18ee8da9c8
 TERMUX_PKG_DEPENDS="libc++"
 TERMUX_PKG_BREAKS="libdb-dev"
