diff --git a/packages/less/build.sh b/packages/less/build.sh
index 1111111111111111111111111111111111111111..2222222222222222222222222222222222222222 100644
--- a/packages/less/build.sh
+++ b/packages/less/build.sh
@@ -6,7 +6,7 @@ TERMUX_PKG_LICENSE_FILE='COPYING, LICENSE'
 TERMUX_PKG_MAINTAINER="Joshua Kahn <tom@termux.dev>"
 TERMUX_PKG_VERSION="704"
-TERMUX_PKG_SRCURL="https://www.greenwoodsoftware.com/less/less-${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SRCURL="https://mirrors.kernel.org/gnu/less/less-${TERMUX_PKG_VERSION}.tar.gz"
 TERMUX_PKG_SHA256=20a0b0a2bb2525fa53c7eee9beb854b4c9cf172eabb209af7020743547bfe9fb
 TERMUX_PKG_DEPENDS="ncurses, pcre2"
 TERMUX_PKG_REPLACES="lazyread"
 TERMUX_PKG_ESSENTIAL=true
